ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -182,3 +182,12 @@ SRC2:
   - source2_phase3_phase1_over_voltage_status
   - source2_over_voltage_status
   - source2_lost_power_alarm
+
+ALM:
```

Comment:
be specific, these are not general alarms. should have a name like FALM for 'failed alarm' or something similar.

Reply:
I see sir. I'll remove the whole ALM

Reply:
Noted sir, I'll remove ALM